sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -12,19 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import multiprocessing
-from typing import Callable, List, Dict, Any
 import inspect
+import multiprocessing
 import textwrap
-import grpc
 import time
+from typing import Any, Callable, Dict, List, Optional
 
-from kubernetes import client, config
+import grpc
```

Comment:
Do we need to move grpc import to `kubeflow.katib` imports ?
Usually, we try to separate internal imports (e.g. `kubeflow.katib`) and external imports (e.g. `time`, `grpc`, etc.)

Reply:
I used [isort](https://pycqa.github.io/isort/) to automate import organization. It's very widely adopted and actually part of the KFP SDK pre-commit hooks, which are executed during CI to ensure code standardization (side note: I'd be happy to setup pre-commit hooks for this repository). What's nice about isort is it takes all of the ambiguity and subjectivity out of import organization and fully automates the process. It typically separates between and alphabetizes standard library imports and non-standard library imports. `grpc` isn't part of the standard library, which is why `isort` puts it in the second block. All that being said, I'm not very opinionated one way or the other and don't mind relocating `grpc` if that's preferable!

Reply:
Sounds good! Issue filed: https://github.com/kubeflow/katib/issues/2187.